t/05-messages/03-errors.t

```diff
@@ -175,4 +175,12 @@ cmp-ok X::OutOfRange.new(
 is-run 'class { method z { $^a } }', :err{ my @lines = $^msg.lines; @lines.grep({ !/'⏏'/ && .contains: '$^a' }) }, :exitcode{.so},
 'Use placeholder variables in a method should yield a useful error message';
 
+# https://github.com/rakudo/rakudo/issues/2921
+is-run 'bleah:(0)', err => { .contains: 'adverb' }, :exitcode{.so},
```

Comment:
I'd test against _«You can't adverb»_ part of the message to make the test more reliable.

Reply:
I think the policy is not to check for specific phrasing as it might change later (to say `You cannot adverb` or `You can not adverb` etc), just checking there is some output to stderr and it contains a word `adverb`.
If you still deem it as a better idea, I'll gladly patch.

Reply:
I don't consider the policy to be that strict and it rather should apply to roast, in my view.

Besides, if wording changes then it's not such a big deal to change the test. A regression might change the error to something totally different in meaning, yet still containing 'adverb' in it. So, yes, I would very much prefer it to be three words.